### ISSUE TITLE
Configure commit lint

### ISSUE THREAD

**Issue by EduardoLopes** [cubos/web-utils#3]

**Comment by EduardoLopes** [comment]
I was thinking, since we'll use squash and merge, we don't really need commit linting, because all commits will be squashed into a single commit when merged into main. BUT, the PR title needs to be semantic, since it's the PR title that is used to the squashed commit message.

**Comment by EduardoLopes** [comment]
of topic: a documentação será primariamente em inglês, e aqui nas issues? iremos discutir em inglês ou português?

**Comment by marcelovicentegc** [comment]
> I was thinking, since we'll use squash and merge, we don't really need commit linting, because all commits will be squashed into a single commit when merged into main. BUT, the PR title needs to be semantic, since it's the PR title that is used to the squashed commit message. c/c @marcelovicentegc

Good point @EduardoLopes ! IMO we should use commit lint to at least keep a semantic pattern on the project scope (not necessarily only on the target branch scope). If, for instance, we wish to explore a branch before/after merge, we'll at least see a specific pattern. What's your take on this? 

Regarding the off topic point, IMO we should primarily use english, but I would say by the end of the day it's up to the dev, really. As I know you're comfortable with english, I'll discuss in english with you. But if others don't feel comfortable with it, I don't see a problem in having a multi-language discussion or discussions in different languages :smile:

**Comment by EduardoLopes** [comment]
I agree! DALE
